src/main/scala/beam/agentsim/infrastructure/taz/TAZTreeMap.scala

```diff
@@ -177,10 +209,10 @@ object TAZTreeMap {
     val quadTreeBounds: QuadTreeBounds = quadTreeExtentFromShapeFile(features)
 
     val tazQuadTree: QuadTree[TAZ] = new QuadTree[TAZ](
-      quadTreeBounds.minx,
-      quadTreeBounds.miny,
-      quadTreeBounds.maxx,
-      quadTreeBounds.maxy
+      quadTreeBounds.minx - 2e4,
```

Comment:
You should turn this into a constant value with a name to better understand its purpose. As it stands I do not understand the reasoning for the addition myself - beyond buying some space.

Reply:
Same with the 2e3 above